Utilities/src/main/java/edu/stanford/hivdb/mutations/Apobec.java

```diff
@@ -113,13 +107,24 @@ public String generateComment() {
 		return comment.toString();
 	}
 
+	static void staticInit() {
```

Comment:
`protected`. I also didn't see the rationale why we should create a `staticInit` function out of `static {}` definition.

Reply:
From the [documentation](https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html), I explored applying the concept that "the advantage of ... methods is that they can be reused later if you need to reinitialize the class variable." Specifically, I wondered whether wrapping the static init block in a callable method would enable me to test it. Of course, I discovered additional complexities (such as reliance on the db class) that made this refactorization a dead end. I decided to leave the wrapper method because of its call-ability. However, since the static block only calls a single line of code (Cachable.setup()), I also agree that this wrapper adds unnecessary bloat to the file without offering any functional benefit. I will change it back. 

That being said, I believe CodonTranslator, for example, could benefit from initializing its static fields in the way the documentation outlines

```java
class Whatever {
    public static varType myVar = initializeClassVariable();
        
    private static varType initializeClassVariable() {

        // initialization code goes here
    }
}
```
but, of course, CodonTranslator is also much more testable that Apobec. So there's no real need to refactor it right now.

Reply:
OK. Now I understand your rationale and I agree with you totally.